centaur/src/main/resources/standardTestCases/cwl_input_typearray/input_typearray.yml

```diff
@@ -0,0 +1,3 @@
+value:
+  class: File
+  path: "centaur/src/main/resources/standardTestCases/cwl_input_typearray/input.txt"
```

Comment:
I think to test this properly we would also want another input that looks the same in the CWL but is input more like:
```
value:
  class: String
  path: "centaur/src/main/resources/standardTestCases/doesnt_exist.txt"
```

In other words - if we provide the `String` version of the input, Cromwell should not try to localize anything and things should still work just fine.

Reply:
Sounds reasonable--added a second input and output to the test!